### PR TITLE
feat: Disable language service for non-Angular projects

### PR DESCRIPTION
**Pull request by kyliau**
This PR disables the language service for a particular project if it is
not an Angular project. The detection mechanism relies on the presence
of `@angular/core/core.d.ts` file in the program.

PR closes https://github.com/angular/vscode-ng-language-service/issues/391

### CODE REVIEW

File: server/src/project_service.ts [hunk 1]
```diff
@@ -173,3 +184,22 @@ export class ProjectService {
...
+ * Return true if the specified `project` contains the Angular core declaration
+ * file.
+ * @param project
+ */
+function isAngularProject(project: ts.server.Project): boolean {
+  if (project.isNonTsProject()) {
+    return false;
+  }
+  for (const fileName of project.getFileNames()) {
+    // TODO: Check behavior on Windows - should be ok, since filepath is
```
Comment (andrius-pra) [inline server/src/project_service.ts]: It works on Windows

File: server/src/project_service.ts [hunk 2]
```diff
@@ -129,21 +129,28 @@ export class ProjectService {
...
         break;
-      case ts.server.ProjectLoadingFinishEvent:
-        this.connection.sendNotification(
-            projectLoadingNotification.finish, event.data.project.projectName);
+      }
+      case ts.server.ProjectLoadingFinishEvent: {
+        const {project} = event.data;
+        const {projectName} = project;
+        this.connection.sendNotification(projectLoadingNotification.finish, projectName);
+        if (!isAngularProject(project)) {
```
Comment (andrius-pra) [inline server/src/project_service.ts]: Suggestion: add check if plugin loaded. ex:
```
if (project.projectKind != ts.server.ProjectKind.Configured) {
  project.disableLanguageService();
}

const globalPlugins = project['plugins'] as {name: string, module: {VERSION: {full: string}}}[];
const plugin = globalPlugins && globalPlugins.find(x => x.name === '@angular/language-service');

if (plugin) {
  const version = plugin.module.VERSION.full;
  this.connection.console.info(`Success: @angular/language-service ${version} loaded`);
} else {
  this.connection.console.error('Failed to load @angular/language-service');
  project.disableLanguageService();
}
```

**Comment by jpike88** [comment]
how's this looking?

**Comment by ayazhafiz** [comment]
@kyliau please check the test error

**Pull request merged**